clients/fides-js/docs/interfaces/Fides.md

```diff
@@ -9,6 +9,8 @@ FidesJS integrations (e.g. `Fides.gtm()`), programmaticaly show the FidesJS
 UI (e.g. `Fides.showModal()`) and more. See the full list of properties below
 for details.
 
+bla bla bla
+
```

Comment:
is this supposed to be in there?

Reply:
Not long term! I added this to prove that it was working. Will remove before merge. But I need a ticket # anyway.